xla/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -5212,7 +5212,7 @@ GetCudnnFusedMHAOperationGraph(
           auto bias_out,
           CreateCudnnBiasTensor(intermediate_ops, intermediate_bmm2_lhs_dims,
                                 intermediate_bmm2_lhs_strides,
-                                bias_descriptor.type(), bmm2_input_tensor,
+                                (*bias_descriptor).type(), bmm2_input_tensor,
```

Comment:
i know this is within the use_bias condition so bias_descriptor will likely not be null, but maybe add a CHECK before this?

Reply:
I added a DCHECK to guard the dereference